flight/targets/sprf3e/fw/pios_board.c

```diff
@@ -0,0 +1,479 @@
+/**
```

Comment:
We do have a tool to clean up flight code, to install it:
`make uncrustify_install`
Then to clean this file:
`make uncrustify_flight FILE=flight/targets/sprf3e/fw/pios_board.c`

I think it should fix these types of things.